File: Python/Armstrong.py
def Armstrong():
    Liste = []

    for i in range(100,1000,1):
        YuzB, OnB, BirB = int(str(i)[2]), int(str(i)[1]), int(str(i)[0])
        YuzKup, OnKup, BirKup = YuzB**3,OnB**3,BirB**3

        if i == YuzKup+OnKup+BirKup:
            Liste.append(i)

    return Liste

print(Armstrong())

File: Java/VizeSoru/src/Armstrong.java
public class Armstrong {
	public static void main(String[] args) {
		
		int YuzB, OnB, BirB, YuzKup, OnKup, BirKup;
		
		System.out.println("100-999 arasýndaki Armstrong sayýlarý:");
		System.out.println("--------------------------------------") ;
		
		for(int i=100;i<1000;i++){
			YuzB = i/100;
			OnB = (i-(YuzB*100))/10;
			BirB = i-(YuzB*100+OnB*10);
			YuzKup = YuzB*YuzB*YuzB;
			OnKup = OnB*OnB*OnB;
			BirKup = BirB*BirB*BirB;
			
			if(i == (YuzKup+OnKup+BirKup))
				System.out.println(i);
		}
	}
}